sdk/keyvault/Azure.Security.KeyVault.Secrets/README.md

```diff
@@ -14,6 +14,13 @@ Install the Azure Key Vault secrets client library for .NET with [NuGet][nuget]:
 dotnet install Azure.Security.KeyVault.Secrets
 ```
 
+You also need to install the Azure Identity library for .NET using [NuGet][nuget]:
+```PowerShell
+dotnet install Azure.Identity
```

Comment:
This is mentioned below on line 42 and only for the samples included below that. It's not *required* to authenticate. Only `Azure.Core` is required, by design, since `TokenCredential` exists therein.

Reply:
Thanks @heaths . I'll close the PR... This is already covered in the Readme.MD I just didn't see it :( Long hours will do this to you